src/main/kotlin/no/nav/syfo/consumer/distribuerjournalpost/JournalpostdistribusjonConsumer.kt

```diff
@@ -0,0 +1,81 @@
+package no.nav.syfo.consumer.distribuerjournalpost
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.json.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import no.nav.syfo.UrlEnv
+import no.nav.syfo.auth.AzureAdTokenConsumer
+import org.slf4j.LoggerFactory
+
+class JournalpostdistribusjonConsumer(urlEnv: UrlEnv, azureAdTokenConsumer: AzureAdTokenConsumer) {
+    private val client: HttpClient
+    private val dokdistfordelingUrl: String
+    private val dokdistfordelingScope: String
+    private val azureAdTokenConsumer: AzureAdTokenConsumer
+
+    private val LOG = LoggerFactory.getLogger("no.nav.syfo.consumer.JournalpostdistribusjonConsumer")
+
+    init {
+        client = HttpClient(CIO) {
```

Comment:
Her kan du bruke httpClient() fra HttpUtil for å fjerne litt boilerplate

Reply:
Kunne muligens byttet ut de fleste eller alle custom httpClients med den fra HttpUtil? Ser der er andre tilfeller i koden her og der